client/src/pages/TripPage/TripPage.js

```diff
@@ -1,14 +1,22 @@
+import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
+import Header from '../../components/Header/Header'
```

Comment:
I find it nice to have an index.js in that way your paths are not always doubled.